homeassistant/components/zwave/__init__.py

```diff
@@ -794,7 +796,7 @@ def value_changed(self):
         # If value changed after device was created but before setup_platform
         # was called - skip updating state.
         if self.hass:
```

Comment:
We are certain that we don't have to schedule an update if self._scheduled_update:

```python
if self.hass and not self._scheduled_update:
```

(yes, we need a check in both places)